plasmapy/addons/__init__.py

```diff
@@ -0,0 +1,28 @@
+__all__ = []
+
+from pkg_resources import (iter_entry_points, resource_filename)
+
+
+def _parse_readme():
+    fname = resource_filename('plasmapy', 'addons/README.md')
+    with open(fname, encoding='utf-8') as f:
+        readme = f.read()
+
+    return readme
+
+
+__readme__ = _parse_readme()
+
+
+# import addon entry points
+for ep in iter_entry_points('plasmapy.addons'):
+    # __dict__[ep.name] = ep.load()
+    globals()[ep.name] = ep.load()
+    __all__.append(ep.name)
+
+try:
+    del ep
+except NameError:
+    pass
```

Comment:
```suggestion
if "ep" in locals():
    del ep
```
This could be a shorter way of doing it.  I think it should be `locals()` and not `globals()`.  The original way is perfectly fine too, just verbose.

Reply:
Since we’re at the module level here, `locals()` and `globals()` are the same thing.

If I can, I usually code by [EAFP](https://docs.python.org/3/glossary.html#term-eafp) instead of [LBYL](https://docs.python.org/3/glossary.html#term-lbyl).  In this case I don’t think it really matters though.

Reply:
Cool, new acronyms!